test/java/src/com/ibm/streamsx/topology/test/api/IsolateTest.java

```diff
@@ -21,54 +23,87 @@
 import com.ibm.streamsx.topology.tester.Tester;
 
 public class IsolateTest {
-    
+    @Ignore
     @Test
-    public void simpleIsolationTest() throws Exception{
+    public void simpleIsolationTest() throws Exception {
         assumeTrue(SC_OK);
         Topology topology = new Topology("isolationTest");
 
         // Construct topology
         TStream<String> ss = topology.strings("hello");
         TStream<String> ss1 = ss.transform(getPEId(), String.class).isolate();
-        TStream<String> ss2 = ss.isolate().transform(getPEId(), String.class).isolate();
-        
+        TStream<String> ss2 = ss.isolate().transform(getPEId(), String.class)
+                .isolate();
+
         Tester tester = topology.getTester();
-        
-        Condition<List<String>> condss1 = tester.stringContents(ss1, "");      
+
+        Condition<List<String>> condss1 = tester.stringContents(ss1, "");
         Condition<List<String>> condss2 = tester.stringContents(ss2, "");
-        
-	// Jenkins seems to fail when running this with a distributed tester.
-	// Not sure why, but the tests pass on my local build. 
-        /*try{
-        StreamsContextFactory
-                .getStreamsContext(StreamsContext.Type.DISTRIBUTED_TESTER)
-                .submit(topology).get(90, TimeUnit.SECONDS);
-        } catch(Exception e){
+
+        // Jenkins seems to fail when running this with a distributed tester.
```

Comment:
The issue is that the test needs to follow the standard pattern of extending TestTopology and then have 'assume' checks that make it only run in distributed. currently Jenkins is not running distributed as it doesn't yet have an instance setup.